### PR TITLE
fix helmfile issue links

### PR DESCRIPTION
**Pull request by yxxhero**
Signed-off-by: yxxhero <aiopsclub@163.com>

fix helmfile issue links

Sorry. It's my mistake.

### CODE REVIEW

File: content/en/blog/2022-04-19-tools-to-manage-helm-declaratively.md [hunk 1]
```diff
@@ -75,8 +75,8 @@ If you want to use a configuration manager with your Helm and Kubernetes configu
...
 [^3]: ArgoCD shells out to the Helm CLI, only to render templates. This has allowed Argo to turn on Helm CLI's OCI feature before it was finished, for the same reason that it can not support Helm features beyond templating. Because of this, OCI is not part of the ArgoCD source architecture.
-[^4]: Helmfile has a custom concept of hooks, not necessarily mapped to Helm hooks. See readme [hooks section](https://github.com/helmfile/helmfile#hooks) and [this issue](https://github.com/helmfile/helmfile/issues/1291) for clarification and work in progress.
-[^5]: Helmfile has experimental OCI support, without explicitly explaining to users that it sets `HELM_EXPERIMENTAL_OCI=1` before shelling out to the Helm CLI. See [#2112](https://github.com/helmfile/helmfile/issues/2112) and [#2111](https://github.com/helmfile/helmfile/issues/2111).
+[^4]: Helmfile has a custom concept of hooks, not necessarily mapped to Helm hooks. See readme [hooks section](https://github.com/helmfile/helmfile#hooks) and [this issue](https://github.com/roboll/helmfile/issues/1291) for clarification and work in progress.
```
Comment (hickeyma) [inline content/en/blog/2022-04-19-tools-to-manage-helm-declaratively.md:78]: https://github.com/helmfile/helmfile#hooks --> https://github.com/roboll/helmfile#hooks

**Review by hickeyma — CHANGES_REQUESTED** [review]
@yxxhero I have included a link to be changed inline. Also, Helmfile link in https://github.com/helm/helm-www/blob/7e2dbeb07f67bdbee1565b520da570a38572f214/content/en/blog/2022-04-19-tools-to-manage-helm-declaratively.md#other-projects needs to be changed to `roboll` also. Please also recheck the whole document to be sure that its not using a fork in a link.

File: content/en/blog/2022-04-19-tools-to-manage-helm-declaratively.md (hunk 1 above)
Comment (yxxhero) [reply content/en/blog/2022-04-19-tools-to-manage-helm-declaratively.md:78]: https://github.com/helmfile/helmfile#hooks is ok. It can access.

Comment (yxxhero) [reply content/en/blog/2022-04-19-tools-to-manage-helm-declaratively.md:78]: only issue link need to fix. what do you think of? Thanks very much.

Comment (yxxhero) [reply content/en/blog/2022-04-19-tools-to-manage-helm-declaratively.md:78]: @hickeyma

**Comment by yxxhero** [comment]
@hickeyma see: https://github.com/roboll/helmfile#please-read-helmfile-has-moved-home

File: content/en/blog/2022-04-19-tools-to-manage-helm-declaratively.md (hunk 1 above)
Comment (hickeyma) [reply content/en/blog/2022-04-19-tools-to-manage-helm-declaratively.md:78]: yes, that's correct

**Review by hickeyma — APPROVED** [review]
My mistake @yxxhero. I saw the fork to https://github.com/helmfile/helmfile/ and thought it was incorrect. I didn't realise that the repo wasn't moved but forked instead. 

Thanks for fixing the issue links.

**Comment by yxxhero** [comment]
@hickeyma Thanks very much.

**Pull request merged**